Change table width CSS in inventory

davidlowryduda opened the pull request:
When I made the inventory records sortable, I used class for which the LMFDB has some CSS. This is a minor tweak to make the width *not* take the whole screen, resolving something John pointed out to me.

JohnCremona commented:
Maybe I am doing something wrong but it looks just the same to me now!  There is a space prefix on every line (if multiline) in the top row only.

davidlowryduda commented:
@JohnCremona Ah, I see. Initially I had thought that you were seeing something width-related since I know the monitor in your office has a vastly wider resolution than my screens. So I switched to a high-resolution viewer and noticed that 100% width was a silly idea.

But now I notice inline spacing. And I think I've corrected it now. Would you mind looking again?

JohnCremona commented:
Fine now -- thanks.  I was only testing on a laptop.
